quipucords/scanner/satellite/utils.py

```diff
@@ -155,15 +152,16 @@ def status(scan_task):
     """
     try:
         return _status6(scan_task)
-    except SatelliteException as sat_error:
-        message = 'Satellite 6 status check failed with error:' \
-            ' "%s". Attempting Satellite 5.' % sat_error
+    except SatelliteException as sat_error:  # pylint: disable=unused-variable # noqa F841
```

Comment:
instead of disabling this check consider removing the variable

(for the future I'd rather have the full traceback instead of only the message that follows)

Reply:
done